components/job/JobList.js

```diff
@@ -56,6 +56,7 @@ const JobList = () => {
           <ul>
             {activeApplications &&
               activeApplications.map((job) => {
+                // TODO: ?
                 return <JobItem job={job} key={job.id} />
```

Comment:
@drewclem This component doesn't exist as far as I can tell. Should we remove any components like this from the jobs page for now?

Reply:
Yeah, that sounds good. I'm not sure how some of these came to be- other than getting left behind while iterating